runbotics/runbotics-orchestrator-ui/src/src-app/components/tables/NotificationTable/NotificationTableComponent.types.ts

```diff
@@ -0,0 +1,17 @@
+import { IUser, UserBot, UserProcess } from 'runbotics-common';
+
+export interface ProcessNotificationRow {
+    id: number;
+    userId: UserProcess['userId'];
+    processId: UserProcess['processId'];
+    user: IUser['email'];
+    subscribedAt: UserProcess['subscribedAt'];
+}
+
+export interface BotNotificationRow {
+    id: number;
+    userId: UserBot['userId'];
+    botId: UserBot['botId'];
+    user: IUser['email'];
+    subscribedAt: UserBot['subscribedAt'];
+}
```

Comment:
it's nice to have an enum for userId, processId, etc. - consider if it's used elsewhere too